### DISCUSSION TITLE
unable to open slave

### DISCUSSION THREAD

**Discussion by GiveMeFox** [ppy/osu#23742] [Q&A] [answered]
Lazer 2023.513.0
6.3.5-arch1-1

![image](https://github.com/ppy/osu/assets/74460065/7ade53f8-6bb8-4513-af55-e8ef4f5c6ac2)

im getting this `ALSA lib pcm_dmix.c:999:(snd_pcm_dmix_open) unable to open slave` and the audio is distorted its the first time this happend to me audio from youtube and other sources sounds fine

**Comment by Axilotl17** [comment]
you wanna send logs?

**Comment by GiveMeFox** [comment]
[runtime.log](https://github.com/ppy/osu/files/11657740/runtime.log)
[network.log](https://github.com/ppy/osu/files/11657741/network.log)
[performance.log](https://github.com/ppy/osu/files/11657742/performance.log)
[input.log](https://github.com/ppy/osu/files/11657743/input.log)
[database.log](https://github.com/ppy/osu/files/11657750/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11657751/legacy-ipc.log)

**Reply by GiveMeFox** [reply, marked as answer]
i just updated to 2023.605.0 and the audio is now normal but i get the same message

**Reply by GiveMeFox** [reply]
ok it seems like the issue resolved itself i no longer get that message
